hermes/wasi-preview1-component-adapter/Cargo.toml

```diff
@@ -0,0 +1,91 @@
+#cspell: words Apisit Ritreungroj wasi bindgen cdylib
```

Comment:
I would rather this package NOT live in the `hermes` subdir.
This is because `hermes` itself is a workspace.
So this, which is an independent workspace is actually a child in the directory hierarchy of another workspace which is confusing.
It would be better if it was back in the `wasm` directory off the root of the repo.  So that there is no ambiguity that it is NOT and never will be part of the hermes workspace.

I would also like the `wasi/wit` files to be in the `wasm` directory and also not as children of the `hermes` directory.

Reply:
We only need the wit files specifically required for the `wasi-preview1-component-adapter`.  The other ones for our specific API's can be located in the other location.
We need to sort out the .wit files, but its a different task.

Reply:
Ahh, got it, so right now all files inside `wit` dir is related to the `wasi-preview1-component-adapter`.